Fenceupdate (#3494)
ArduPilot/ardupilot_wiki: common/source/docs/common-geofencing-landing-page.rst

[/site]

[site wiki="plane"]

> Plane firmware prior to 4.1  supports failsafes generated by a polygonal GeoFence (boundaries described by latitude/longitude and altitude) to prevent the vehicle from traveling higher or further than desired, or into unwanted areas. See :ref:`plane:geofencing`

Reviewer: ```suggestion
Plane firmware prior to 4.1 supports failsafes generated by a polygonal GeoFence (boundaries described by latitude/longitude and altitude) to prevent the vehicle from traveling higher or further than desired, or into unwanted areas. See :ref:`plane:geofencing`
```

Author: fixed

> Firmware versions 4.1 and later, support a home based Cylindrical and Polgonal or Cylinderical Inclusion and/or Exclusion regions. Inclusion and Exclusion Fences are defined easily in Mission Planner under the PLAN screen and loaded to the autopilot, using the FENCES item in the drop down box, just like Missions or Rally Points are planned.

Reviewer: Spelling. 

```suggestion
Firmware versions 4.1 and later, support a home based Cylindrical and Polygonal or Cylindrical Inclusion and/or Exclusion regions. Inclusion and Exclusion Fences are defined easily in Mission Planner under the PLAN screen and loaded to the autopilot, using the FENCES item in the drop down box, just like Missions or Rally Points are planned.
```

Author: fixed

> Firmware versions 4.1 and later, support a home based Cylindrical and Polgonal or Cylinderical Inclusion and/or Exclusion regions. Inclusion and Exclusion Fences are defined easily in Mission Planner under the PLAN screen and loaded to the autopilot, using the  FENCES item in the drop down box, just like Missions or Rally Points are planned.

Reviewer: ```suggestion
Firmware versions 4.1 and later, support a home based Cylindrical and Polgonal or Cylinderical Inclusion and/or Exclusion regions. Inclusion and Exclusion Fences are defined easily in Mission Planner under the PLAN screen and loaded to the autopilot, using the FENCES item in the drop down box, just like Missions or Rally Points are planned.
```

    Inclusion/Exclusion Fence Failsafe <common-polygon_fence>
[/site]
[site wiki="plane"]
    Cylindrical Fence Failsafe (version 4.1 and later) <common-ac2_simple_geofence>

>     Inclusion/Exclusion Fence Failsafe (version 4.1 and later)<common-polygon_fence>

Reviewer: Not familiar with the syntax here, is a space required after the close bracket and open arrow?
here --> `... later)<common-...`

```suggestion
    Inclusion/Exclusion Fence Failsafe (version 4.1 and later) <common-polygon_fence>
```

Author: no...spaces ignored
